apps/staking/app/stake/OpenNodes.tsx

```diff
@@ -31,6 +31,7 @@ export default function OpenNodes() {
     return new Set(contracts.map(({ pubkey_bls }) => pubkey_bls));
   }, [contracts]);
 
+  // biome-ignore lint/correctness/useExhaustiveDependencies: On mount
```

Comment:
I wonder if it is worth you making your own useMount hook? Would mean you only need this disable in one place.

Reply:
I agree, will do as a separate PR next